api_fhir_r4/serializers/communicationSerializer.py

```diff
@@ -20,4 +20,9 @@ def create(self, validated_data):
         from core import datetime
         copied_data['feedback_date'] = datetime.datetime.now()
         obj = Feedback.objects.create(**copied_data)
+        imis_claim = Claim.objects.get(id=claim, validity_to__isnull=True)
+        imis_claim.feedback_status = Claim.FEEDBACK_SELECTED
```

Comment:
I think it should be `DELIVERED`

Reply:
Why? When we discussed you said that it needed to be SELECTED (4) so it will appear in the list.

Indeed, if it needs to be delivered I'll do, but let's be sure what it should be :grin:. Do you have any ref to share?

Reply:
There was something lost in translation.
My understanding is that the `Claim`'s `feedbackStatus` must be `SELECTED (4)` for the feedback **request** to be shown in the application.
When the feedback is sent to the backend, the request is therefore fulfilled. Thus, there is no more feedback request, the `feedbackStatus` should be `DELIVERED (8)`.

@tdethier could you confirm my understanding is correct?